### DISCUSSION TITLE
UnavailableInvalidChannel: HTTP 403 FORBIDDEN for channel public <https://conda.software.inl.gov/public>

### DISCUSSION THREAD

**Discussion by LuHaiTao11** [idaholab/moose#28567] [Q&A Getting Started]
### Check these boxes if you have followed the posting rules.

- [X] I have consulted the [Posting Guidelines](https://github.com/idaholab/moose/discussions/18270).
- [X] I have searched the [Discussions Forum](https://github.com/idaholab/moose/discussions) and [MOOSE Framework Troubleshooting](https://mooseframework.inl.gov/help/troubleshooting.html) and have not found what I was looking for
- [X] Q&A Getting Started is the most appropriate category for my question (trouble installing, beginner user, ...)

### Issue or question about MOOSE

Failed to create the conda environment fro Moose.
UnavailableInvalidChannel: HTTP 403 FORBIDDEN for channel public <https://conda.software.inl.gov/public>

**Comment by GiudGiud** [comment]
hello

Can you please try again now? There are network flukes

**Reply by Aayush-T** [reply]
It is still happening. So, I was trying to make some other environments for my personal work like 2-3 days agao and since this "https://conda.software.inl.gov/public" is already added, it gave the same error then too. I had to disable it for my other environment to run. 
I tested just now, and it is still giving the same error. 
(base) xyz@xyz-abc:~$ conda create -n moose moose-dev=2024.08.30 mpich
Channels:
 - https://conda.software.inl.gov/public
 - conda-forge
Platform: linux-64
Collecting package metadata (repodata.json): failed

UnavailableInvalidChannel: HTTP 403 FORBIDDEN for channel public <https://conda.software.inl.gov/public>

The channel is not accessible or is invalid.

You will need to adjust your conda configuration to proceed.
Use `conda config --show channels` to view your configuration's current state,
and use `conda config --show-sources` to view config file locations.

(base) xyz@xyz-abc:~$ conda config --show-sources
==> /home/xyz/miniforge/.condarc <==
channels:
  - conda-forge

==> /home/xyz/.condarc <==
channels:
  - https://conda.software.inl.gov/public
  - conda-forge


So, I looked into it using curl:
*** curl https://conda.software.inl.gov/public/ ***

<!DOCTYPE html><html lang="en-US"><head><title>Just a moment...</title><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=Edge"><meta name="robots" content="noindex,nofollow"><meta name="viewport" content="width=device-width,initial-scale=1"><style>*{box-sizing:border-box;margin:0;padding:0}html{line-height:1.15;-webkit-text-size-adjust:100%;color:#313131}button,html{font-family:system-ui,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica Neue,Arial,Noto Sans,sans-serif,Apple Color Emoji,Segoe UI Emoji,Segoe UI Symbol,Noto Color Emoji}body{display:flex;flex-direction:column;height:100vh;min-height:100vh}body.no-js .loading-spinner{visibility:hidden}body.theme-dark{background-color:#222;color:#d9d9d9}body.theme-dark a{color:#fff}body.theme-dark a:hover{color:#ee730a;text-decoration:underline}body.theme-dark .lds-ring div{border-color:#999 transparent transparent}body.theme-dark .font-red{color:#b20f03}body.theme-dark .pow-button{background-color:#4693ff;color:#1d1d1d}body.theme-dark #challenge-success-text{background-image:url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIzMiIgaGVpZ2h0PSIzMiIgZmlsbD0ibm9uZSIgdmlld0JveD0iMCAwIDI2IDI2Ij48cGF0aCBmaWxsPSIjZDlkOWQ5IiBkPSJNMTMgMGExMyAxMyAwIDEgMCAwIDI2IDEzIDEzIDAgMCAwIDAtMjZtMCAyNGExMSAxMSAwIDEgMSAwLTIyIDExIDExIDAgMCAxIDAgMjIiLz48cGF0aCBmaWxsPSIjZDlkOWQ5IiBkPSJtMTAuOTU1IDE2LjA1NS0zLjk1LTQuMTI1LTEuNDQ1IDEuMzg1IDUuMzcgNS42MSA5LjQ5NS05LjYtMS40Mi0xLjQwNXoiLz48L3N2Zz4=)}body.theme-dark #challenge-error-text{background-image:url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIzMiIgaGVpZ2h0PSIzMiIgZmlsbD0ibm9uZSI+PHBhdGggZmlsbD0iI0IyMEYwMyIgZD0iTTE2IDNhMTMgMTMgMCAxIDAgMTMgMTNBMTMuMDE1IDEzLjAxNSAwIDAgMCAxNiAzbTAgMjRhMTEgMTEgMCAxIDEgMTEtMTEgMTEuMDEgMTEuMDEgMCAwIDEtMTEgMTEiLz48cGF0aCBmaWxsPSIjQjIwRjAzIiBkPSJNMTcuMDM4IDE4LjYxNUgxNC44N0wxNC41NjMgOS41aDIuNzgzem0tMS4wODQgMS40MjdxLjY2IDAgMS4wNTcuMzg4LjQwNy4zODkuNDA3Ljk5NCAwIC41OTYtLjQwNy45ODQtLjM5Ny4zOS0xLjA1Ny4zODktLjY1IDAtMS4wNTYtLjM4OS0uMzk4LS4zODktLjM5OC0uOTg0IDAtLjU5Ny4zOTgtLjk4NS40MDYtLjM5NyAxLjA1Ni0uMzk3Ii8+PC9zdmc+)}body.theme-light{background-color:#fff;color:#313131}body.theme-light a{color:#0051c3}body.theme-light a:hover{color:#ee730a;text-decoration:underline}body.theme-light .lds-ring div{border-color:#595959 transparent transparent}body.theme-light .font-red{color:#fc574a}body.theme-light .pow-button{background-color:#003681;border-color:#003681;color:#fff}body.theme-light #challenge-success-text{background-image:url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIzMiIgaGVpZ2h0PSIzMiIgZmlsbD0ibm9uZSIgdmlld0JveD0iMCAwIDI2IDI2Ij48cGF0aCBmaWxsPSIjMzEzMTMxIiBkPSJNMTMgMGExMyAxMyAwIDEgMCAwIDI2IDEzIDEzIDAgMCAwIDAtMjZtMCAyNGExMSAxMSAwIDEgMSAwLTIyIDExIDExIDAgMCAxIDAgMjIiLz48cGF0aCBmaWxsPSIjMzEzMTMxIiBkPSJtMTAuOTU1IDE2LjA1NS0zLjk1LTQuMTI1LTEuNDQ1IDEuMzg1IDUuMzcgNS42MSA5LjQ5NS05LjYtMS40Mi0xLjQwNXoiLz48L3N2Zz4=)}body.theme-light #challenge-error-text{background-image:url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIzMiIgaGVpZ2h0PSIzMiIgZmlsbD0ibm9uZSI+PHBhdGggZmlsbD0iI2ZjNTc0YSIgZD0iTTE2IDNhMTMgMTMgMCAxIDAgMTMgMTNBMTMuMDE1IDEzLjAxNSAwIDAgMCAxNiAzbTAgMjRhMTEgMTEgMCAxIDEgMTEtMTEgMTEuMDEgMTEuMDEgMCAwIDEtMTEgMTEiLz48cGF0aCBmaWxsPSIjZmM1NzRhIiBkPSJNMTcuMDM4IDE4LjYxNUgxNC44N0wxNC41NjMgOS41aDIuNzgzem0tMS4wODQgMS40MjdxLjY2IDAgMS4wNTcuMzg4LjQwNy4zODkuNDA3Ljk5NCAwIC41OTYtLjQwNy45ODQtLjM5Ny4zOS0xLjA1Ny4zODktLjY1IDAtMS4wNTYtLjM4OS0uMzk4LS4zODktLjM5OC0uOTg0IDAtLjU5Ny4zOTgtLjk4NS40MDYtLjM5NyAxLjA1Ni0uMzk3Ii8+PC9zdmc+)}body.theme-light.feedback-report{border:1px solid #959595}body.feedback-report{border-radius:5px}a{background-color:transparent;color:#0051c3;text-decoration:none;transition:color .15s ease}a:hover{color:#ee730a;text-decoration:underline}.main-content{margin:8rem auto;max-width:60rem;padding-left:1.5rem;padding-right:1.5rem;width:100%}.main-content .spacer{margin:2rem 0}.main-content .loading-spinner{height:76.391px}.feedback-content{align-content:space-between;display:inline-grid;height:100vh;margin:0;padding:0}.feedback-content .spacer{margin:0}.heading-favicon{height:2rem;margin-right:.5rem;width:2rem}@media (width <= 720px){.main-content{margin-top:4rem}.heading-favicon{height:1.5rem;width:1.5rem}.feedback-content{margin-top:0}}.main-wrapper{align-items:center;display:flex;flex:1;flex-direction:column}.feedback-report .main-wrapper{min-height:100vh}.font-red{color:#b20f03}.h1{font-size:2.5rem;font-weight:500;line-height:3.75rem}.h2{font-weight:500}.core-msg,.h2{font-size:1.5rem;line-height:2.25rem}.body-text,.core-msg{font-weight:400}.body-text{font-size:1rem;line-height:1.25rem}@media (width <= 720px){.h1{font-size:1.5rem;line-height:1.75rem}.h2{font-size:1.25rem}.core-msg,.h2{line-height:1.5rem}.core-msg{font-size:1rem}}#challenge-error-text{background-image:url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIzMiIgaGVpZ2h0PSIzMiIgZmlsbD0ibm9uZSI+PHBhdGggZmlsbD0iI2ZjNTc0YSIgZD0iTTE2IDNhMTMgMTMgMCAxIDAgMTMgMTNBMTMuMDE1IDEzLjAxNSAwIDAgMCAxNiAzbTAgMjRhMTEgMTEgMCAxIDEgMTEtMTEgMTEuMDEgMTEuMDEgMCAwIDEtMTEgMTEiLz48cGF0aCBmaWxsPSIjZmM1NzRhIiBkPSJNMTcuMDM4IDE4LjYxNUgxNC44N0wxNC41NjMgOS41aDIuNzgzem0tMS4wODQgMS40MjdxLjY2IDAgMS4wNTcuMzg4LjQwNy4zODkuNDA3Ljk5NCAwIC41OTYtLjQwNy45ODQtLjM5Ny4zOS0xLjA1Ny4zODktLjY1IDAtMS4wNTYtLjM4OS0uMzk4LS4zODktLjM5OC0uOTg0IDAtLjU5Ny4zOTgtLjk4NS40MDYtLjM5NyAxLjA1Ni0uMzk3Ii8+PC9zdmc+);padding-left:34px}#challenge-error-text,#challenge-success-text{background-repeat:no-repeat;background-size:contain}#challenge-success-text{background-image:url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIzMiIgaGVpZ2h0PSIzMiIgZmlsbD0ibm9uZSIgdmlld0JveD0iMCAwIDI2IDI2Ij48cGF0aCBmaWxsPSIjMzEzMTMxIiBkPSJNMTMgMGExMyAxMyAwIDEgMCAwIDI2IDEzIDEzIDAgMCAwIDAtMjZtMCAyNGExMSAxMSAwIDEgMSAwLTIyIDExIDExIDAgMCAxIDAgMjIiLz48cGF0aCBmaWxsPSIjMzEzMTMxIiBkPSJtMTAuOTU1IDE2LjA1NS0zLjk1LTQuMTI1LTEuNDQ1IDEuMzg1IDUuMzcgNS42MSA5LjQ5NS05LjYtMS40Mi0xLjQwNXoiLz48L3N2Zz4=);padding-left:42px}.text-center{text-align:center}.pow-button{background-color:#0051c3;border:.063rem solid #0051c3;border-radius:.313rem;color:#fff;font-size:.875rem;line-height:1.313rem;margin:2rem 0;padding:.375rem 1rem;transition-duration:.2s;transition-property:background-color,border-color,color;transition-timing-function:ease}.pow-button:hover{background-color:#003681;border-color:#003681;color:#fff;cursor:pointer}.footer{font-size:.75rem;line-height:1.125rem;margin:0 auto;max-width:60rem;padding-left:1.5rem;padding-right:1.5rem;width:100%}.footer-inner{border-top:1px solid #d9d9d9;padding-bottom:1rem;padding-top:1rem}.clearfix:after{clear:both;content:"";display:table}.clearfix .column{float:left;padding-right:1.5rem;width:50%}.diagnostic-wrapper{margin-bottom:.5rem}.footer .ray-id{text-align:center}.footer .ray-id code{font-family:monaco,courier,monospace}.core-msg,.zone-name-title{overflow-wrap:break-word}@media (width <= 720px){.diagnostic-wrapper{display:flex;flex-wrap:wrap;justify-content:center}.clearfix:after{clear:none;content:none;display:initial;text-align:center}.column{padding-bottom:2rem}.clearfix .column{float:none;padding:0;width:auto;word-break:keep-all}.zone-name-title{margin-bottom:1rem}}.loading-spinner{height:76.391px}.lds-ring{display:inline-block;position:relative}.lds-ring,.lds-ring div{height:1.875rem;width:1.875rem}.lds-ring div{animation:lds-ring 1.2s cubic-bezier(.5,0,.5,1) infinite;border:.3rem solid transparent;border-radius:50%;border-top-color:#313131;box-sizing:border-box;display:block;position:absolute}.lds-ring div:first-child{animation-delay:-.45s}.lds-ring div:nth-child(2){animation-delay:-.3s}.lds-ring div:nth-child(3){animation-delay:-.15s}@keyframes lds-ring{0%{transform:rotate(0)}to{transform:rotate(1turn)}}@media screen and (-ms-high-contrast:active),screen and (-ms-high-contrast:none){.main-wrapper,body{display:block}}.rtl .heading-favicon{margin-left:.5rem;margin-right:0}.rtl #challenge-success-text{background-position:100%;padding-left:0;padding-right:42px}.rtl #challenge-error-text{background-position:100%;padding-left:0;padding-right:34px}.challenge-content .spacer{margin:2rem 0}.challenge-content .loading-spinner{height:76.391px}@media (prefers-color-scheme:dark){body{background-color:#222;color:#d9d9d9}body a{color:#fff}body a:hover{color:#ee730a;text-decoration:underline}body .lds-ring div{border-color:#999 transparent transparent}body .font-red{color:#b20f03}body .pow-button{background-color:#4693ff;color:#1d1d1d}body #challenge-success-text{background-image:url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIzMiIgaGVpZ2h0PSIzMiIgZmlsbD0ibm9uZSIgdmlld0JveD0iMCAwIDI2IDI2Ij48cGF0aCBmaWxsPSIjZDlkOWQ5IiBkPSJNMTMgMGExMyAxMyAwIDEgMCAwIDI2IDEzIDEzIDAgMCAwIDAtMjZtMCAyNGExMSAxMSAwIDEgMSAwLTIyIDExIDExIDAgMCAxIDAgMjIiLz48cGF0aCBmaWxsPSIjZDlkOWQ5IiBkPSJtMTAuOTU1IDE2LjA1NS0zLjk1LTQuMTI1LTEuNDQ1IDEuMzg1IDUuMzcgNS42MSA5LjQ5NS05LjYtMS40Mi0xLjQwNXoiLz48L3N2Zz4=)}body #challenge-error-text{background-image:url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIzMiIgaGVpZ2h0PSIzMiIgZmlsbD0ibm9uZSI+PHBhdGggZmlsbD0iI0IyMEYwMyIgZD0iTTE2IDNhMTMgMTMgMCAxIDAgMTMgMTNBMTMuMDE1IDEzLjAxNSAwIDAgMCAxNiAzbTAgMjRhMTEgMTEgMCAxIDEgMTEtMTEgMTEuMDEgMTEuMDEgMCAwIDEtMTEgMTEiLz48cGF0aCBmaWxsPSIjQjIwRjAzIiBkPSJNMTcuMDM4IDE4LjYxNUgxNC44N0wxNC41NjMgOS41aDIuNzgzem0tMS4wODQgMS40MjdxLjY2IDAgMS4wNTcuMzg4LjQwNy4zODkuNDA3Ljk5NCAwIC41OTYtLjQwNy45ODQtLjM5Ny4zOS0xLjA1Ny4zODktLjY1IDAtMS4wNTYtLjM4OS0uMzk4LS4zODktLjM5OC0uOTg0IDAtLjU5Ny4zOTgtLjk4NS40MDYtLjM5NyAxLjA1Ni0uMzk3Ii8+PC9zdmc+)}}</style><meta http-equiv="refresh" content="390"></head><body class="no-js"><div class="main-wrapper" role="main"><div class="main-content"><noscript><div id="challenge-error-title"><div class="h2"><span id="challenge-error-text">Enable JavaScript and cookies to continue</span></div></div></noscript></div></div><script>(function(){window._cf_chl_opt={cvId: '3',cZone: "conda.software.inl.gov",cType: 'managed',cNounce: '30567',cRay: '8bfccfedbb728403',cHash: '076a9cc0f3fd1cf',cUPMDTk: "\/public\/?__cf_chl_tk=iUDzdmjyl4LUztmasR8FlMCZYaUEIQfc0ToGoKl1r28-1725776212-0.0.1.1-4330",cFPWv: 'b',cTTimeMs: '1000',cMTimeMs: '390000',cTplV: 5,cTplB: 'cf',cK: "",fa: "\/public\/?__cf_chl_f_tk=iUDzdmjyl4LUztmasR8FlMCZYaUEIQfc0ToGoKl1r28-1725776212-0.0.1.1-4330",md: "IrWjSLSskt0a5boV24SutZ1x3jqGKt_ETQodB0aqG7I-1725776212-1.1.1.1-29UOcU_caFXz2jQBxa5_3rRMX4SY9RcSUCgzGDJDL.9WV87noKYb3cJhHxF.opGZO.6qBb8.WCT2S.D6RKinr5tmEGsDlaeGEY9c.mrxNcsuEB_mBi0hfjbZW4wLI0HjyrwfSX8Gft93fn8wU1jtYN9mDkMuYI2P8H9upoBRAei7OXDt6gix7X8InNfoi14twx5l7nTH5JPii9VOzAEisPD7NqgDUT61U1JFxQaFfySxlMrgywuEf8R990EUfTDP9kVs3PaE0VESmm1tgCGWv7HxBC3BnwldTJaAd.AcLg8cFcwisXAsbD7xrcYze91B48nmNDOy68qmTf1aT_NHS9cMObkfbpX7Tow0hLWGjLlrtVdLr86QPoUv6giMW3JwhVlN0i8I1SwPZE0mAnajCFs5L20a0K6joIaFfguivZ1lejg35EpNHFpNcO3BoD9.WpRjmyEnJGOKpVxLpA8jE8QJyNlA4fgLmoU5RwhOYC.WLMop6qn__WSaeXGZJR7gyUeE5CDqfs0sTDJjQtIg8Oj9vxv9f34EwXoU5GEIn6Z64XWOV5jyw7wF25zfb0eElMq5PGpuY1GMvJuqIjyMWLq9P.UcYzaFOcy_Jfa42H3vlC1Eotv9.mLfQaSGxm3WYnTnksTpm4xmZ4vgPfjqSVd101bttke2LlCsBNEVj3jHXxJSpvzFbeUSxRUbvdIwiqsjOEMW8jZLJI1R4hc7tYpoFqmft51laynjPs2hiXGqojY9SAHs5yF6ghJgUSvfT1OYNKtXqU_r6BZ2lt87F75GfLc5KdCuM_M7V0kA8FuZRK9bbWrrk8_QS8RGkFqoOaT_EO1rv6Lwweb3E2Fo41TP1IakeSgHnRXq2UFn1XGLrzm.DKBb9ndTuTK1CyfRj6wN8qGjONnMag4HvbYrJXhmtMHPNhJGG2qXSjYZ0GLoRfgP15pzP3X3eqmHLxENjC_Z2v64Kr8cihZTPKNuKp1eqhOe5R_doh26h_jcLh9VBRJeZpJATXFVU_._7BrWtJIra4WT0H99OeJJvqDYnlWIc8MEP2DGMNFsd4a0..M0OcEcjWf6XC5iMieoNp1BQHYvRZj9RJ9YPIpYu5niwqSPLrfIVXcHBNq_vwCamKQ8BcGnAHO61w3aHOF4JKDpneAHtsPUffrSpW.Ep7P76Jp.D9duQaG0zBL_Z30wqqLJ3svBujCtHzomurTXg62XDofVKedSwC3Mhlq3Ro4CGCzNSJGqI5dQ3E0JBNsHl9Vp54qd0rI1wmg.r72oobKn954MmNxql8pY5DTBdKkq_HqO79ElPMm.kptMAtb1igkEAwBrvaMR94LZ4tyCZXacmxglI9ICiTYbcyE_f3cr04NNJ9GR12.BXHNyea_8WW3owfPWP3JMC5cZGaS2i62Tq_QfAzHDfDx5Wj05bD2oPg72bwaHqd5L6CdeNCI63hzBaNXPOiKgIObZ9os6cnoEnM0Qg3gRMRYw29ymmKeMgySj8n6ARgYFOHqfwQYIaOcrnD9kXnMCWnjKYSlSbisszT.xEG1grwmJz_wnhWokoaPF.3VDTvGlxnbcWKsBUui8WaP1LlOCXCPu0tb6cHlUonSm0.eKnHhso_R_sY.kebCVEed2nDI00BrTaXOhzaTBKGt_my4_ANA4rIHtWyWFiTbvWHCLbCseNKGTekNzAovwV37vxc_4_iEziPcAWFFdNi8Ee4TrvFVb7dbWZQyvWbSWD2YAtRWhGGHAowryjnYP5TR8yfJOW26Q8vx588nSxGUGN_WnJ8HhZU1s6JNJ5ke804mtRsI1pTN7V8hpxetxG5l_aMVzmpdm6Yu0DZ0Lc0OBy5KoBovsqdsE6HD0vxsbCMzoe_lxqhS.PcVKK_tEDf7VKvPZozvdYdtJYDFvC_wjlTcP53SWgiKT3M6qfdYhmkjcCOHFovRbU7KC4w",mdrd: "IaBlzBlylcxexufERpTghFlu3U2RxqT.Wlq9v7HDFgs-1725776212-1.1.1.1-1QbXke.IEhvbr7keyTxTcbpeHi0VbNx2vtZjglOePv.uFfCQnw.aZmXzduWPrRVZlFrbo7MabcLs.x9Z82jes4KvW_i7IjtTVbQDiuQOBTXkHGVS7mqJkvWRO845lcPncWVs6GgbNwbYYOn5WjO._flvwJGct8z_cRL2jOTjbNF5yDS.z0yXAhr28D66.Gh1GF9cKfYwowf340CX7QVOkvQ5Nfo.Xn7puJb4CZTfmEnzoVPqiM3NG4BiJsRPoa5cNAzihLQkh8AsDV8LjMhpnEPdT1xcBDlf6c8su2_PI3w8zGy3E0KWVW9To2JN2_ZOCddtli__hBfO7yKIo5dc_LVVN_cX78RgtaaS8BIXijQYo1xwNnoGkOUxO_QLQxwoxlwkKjswTKJYMfkZlMOloqM39peN86Y2XFpOZmDxCm3AbniJHzcKmCwQXVZoFPPF0cMVXEry1c_GOtryBP8QDEc9Ok1HFy.UbNwnmv1bVau9k8uUCgbS4LDHAMwUbljKGbfXR0EUnFVDCLjq3UgcuIEmmIBfi6OFOKPBR7WnJBaJLGpOV0QGcyDH.ms33nMFBdWIg.7JZTKn_4Bv8rkHvNQ8DtL7ifCC_aDgWcZQd0AAXAIN3vMWmsVgSbxYwjVSfiDyDCXYSysvHg0aQKQuEJ9t5zUksb4e1E5dSesp4yi2n.h9gPbFNReaj0MV2HiHFJ3GoSMqpAyU_aX2KzoulWzVe9pIwdYMEDr0B03c0Ly_Cy8ZZSpk8JCOSxQTuQ7NelHlEGE8YzWbhv9zfVSa6OGA1TnoXS6ikTJF860eu41Wtc2hkNBjIO71b4jb7RdcZY0waVDNZ8F2SmQBdAakkz2inMbQGciXa8eYkQ41O5sCo7JZ5uVA_1V0f2J9OoKeWxGWekv95ZBKnVNnSKqRrc9vnVJEiVSbzgOqZn3JxhO3dqRRGtn0KKbOKwMNDHB0295d4AY1xAzRUmj94_m9lyHsQxzzmQqnESKlynUvlqyK4cthctK2SeLzaXP65tm5UnzCO3cEM6pu8A39_pTNxrchiC4aM4WwAwgBRR1TNtCbqyuSanFlzv3CcdQBtJcehnu_n38B5KmOBVWgUK.KlfCq.1oMWQWf2dmHvT9be2nEGKOTLYgATr49xjFj2q7fEHJHnnDh0iDCyvMMjpOYR5dwArV26x19z2Wg_GatTYfi4RBRJlUwuxHs6njGV54PgoxZ9HVIK0TaTod6tO6gCClgkDI__tBq0n6cEJRoJutn3vzweZH0MD6kGL23J50kgwe1StUe03MHfjRtUQHJcXfFEwBEiYJOYbO_yoZugvukH3dkz9st1aN_2Gx5ayzPRWD5NcyOrWyKxrEAVmC7wGN4sxw97ysnFbPv9PGQH2qgW_paeslq9ValmpR9ukIdlGMjlzNk3mNYRJI9wB9Qwn6cGjN8bKn9r1cxInU.v6sc_XHHcEFLlO8mR116rLR4pqnE1mthF9O0s2d4KkZFLNyNRnOPDtXblTGQ0R9duMRuOSsLbpX.5L4fByF_8yPXl9BQLFlidRm5JN2m61TGUjPpXFLuG66PbHGIXwk_j.lr7mxRmKmUR5trsiz9Ak5gkxBSWIKrAUs9E5PxhOMtKWX3OqHjMpM8FMZvorHGizhpRP0t1Ay6Tu1fOPok_tmIq9bxrncG7a6a7V6Di.ort12Vr4d2kiw2y6gE0QXWCj7iPvczZB3jHeBF3zgnykQwa2CF.YfdmHJ1cnLN72Gk24H.qCfC3GcV.W_qfs7f4nEt2FBppwsSrKp_uJ2yHgVugE8425vfOeHqa6POibzgNlCyS9O2IF5a48ME9ZCZrOLh.LRTMfyO8Id0WeuNtdu_74SAs7s3qzxxlSiiFy42gIiQrUQyIo9mC.D5fRCwlc5ZJjb1AS3Tcf_t6Q2jPoQ5H5C3CCtIxraK6wFxXgtLu..x1JFPxAnAfya0_dAomLNo.mwBSauSCS2Q7NwTmj09BCqV5cFtvIADC9VSIddyNFgCnIMQLbnUlTvitVwSFqvnoQ9ym9rzOaqd96BOkib29BRGzeMG1CzCwM3YMiQmVGboBnahlyw_3XinMsqO1gwxWoFkctwk.VVqWiQrUXmeQxMA496DyZb7Y6BxW4OYhO3hsnva7b60qvtlikZWAYdhy_e3OVEhwVy1f00MmrZJfaoj_ZBLaOYGzuKa0kH41XQFAkA0lFyf2s1t7POZteJNYwEyju6e_txyExzWy21x9RQTehhICIgyEOj7W.Vl3g",cRq: {ru: 'aHR0cHM6Ly9jb25kYS5zb2Z0d2FyZS5pbmwuZ292L3B1YmxpYy8=',ra: 'Y3VybC83LjgxLjA=',d: 'DE+t2zmm0qisr2VVia97V7OGt8dZKWvSBnr1OsrG/2ee3hW7EMwD+uc5Z9dk7xQQrmzuoevlPIIJY6KY2cS6pZ/3NMGoEcskOzBSCCz55wIpwNjsh0bJnzY1GyhsXOdexc1QqIVs8db6guHZL1GWc/cK9hVCn1Y2d75CzL/uP07otOQ/zCTQBr/joK2T7MPykTLa6lBOHmx1UXv1H9ysUhZ23YXE1yDFbje7EGdOKwHMZ5gLNy9R2n7m37PikJukRtQ8eVc2jhwZfOMrqEoh6XWF6Mz2XfaloAAG9JB0dpbNTJgTwQmcS9SkdFIsuUr2FXscH+02Di1dConZ8mx5kn5jXoeRr/9Fq6ItJtc0NEBmCEhU60ouQNOV3szCsrEi4qYwz26/+HILa3RcEIXIYC7LZ6CuQrmZ3I0oUmbigjREmXuzfpQdMrShHal03OuQxFTSjUDwAYwa6gfCeLEWfMuy88QhsQGXdsMCAEQggP1OyBCmT8Mm1Yc9SgHLUUkZ',t: 'MTcyNTc3NjIxMi4wMDAwMDA=',cT: Math.floor(Date.now() / 1000),m: 'RlH9MxGBN+cgCuuwxo5+FYCAr6B0KH1kcGfr4QO8Nrc=',i1: 'Juf1TdmUJF0BWUgrE4qczQ==',i2: 'iEJUJTFASrBYmupQ1EhljA==',zh: 'uLxisngW/bb3qPDdknsJSf2WthC5mbPxkv0Cm/TWRDA=',uh: 'XtPohTwN9ruoNwJrCbocmA5soW1XPnkxU/yVFmBLOQo=',hh: 'sAHX5K2F4U/P8Yh7oPCuAZ/YhAp4v25WyS17SbSHaDU=',}};var cpo = document.createElement('script');cpo.src = '/cdn-cgi/challenge-platform/h/b/orchestrate/chl_page/v1?ray=8bfccfedbb728403';window._cf_chl_opt.cOgUHash = location.hash === '' && location.href.indexOf('#') !== -1 ? '#' : location.hash;window._cf_chl_opt.cOgUQuery = location.search === '' && location.href.slice(0, location.href.length - window._cf_chl_opt.cOgUHash.length).indexOf('?') !== -1 ? '?' : location.search;if (window.history && window.history.replaceState) {var ogU = location.pathname + window._cf_chl_opt.cOgUQuery + window._cf_chl_opt.cOgUHash;history.replaceState(null, null, "\/public\/?__cf_chl_rt_tk=iUDzdmjyl4LUztmasR8FlMCZYaUEIQfc0ToGoKl1r28-1725776212-0.0.1.1-4330" + window._cf_chl_opt.cOgUHash);cpo.onload = function() {history.replaceState(null, null, ogU);}}document.getElementsByTagName('head')[0].appendChild(cpo);}());</script></body></html>

Also, when we try to open the website, we have to solve Cloudflare verifier:
![image](https://github.com/user-attachments/assets/84680b68-38ea-4644-833f-b057bfb175cf)

So, this is happening due to a Cloudflare protection layer on the MOOSE public Conda channel. When accessing it via the browser, we are completing the JavaScript-based security challenge (as shown above, which checks for bots), but when conda or curl tries to access the channel, it cannot complete this challenge and therefore results in a 403 "Forbidden" error.

**Reply by GiudGiud** [reply]
This is a recent change. 

Tagging @milljm @loganharbour for awareness

A tentative workaround for now is to download the (moose-dev & dependencies) packages from your browser then install offline
```
>conda install --offline <path to package>
```
you ll still want to be online to install all the dependencies from conda-forge

**Reply by MaheshTummala17** [reply]
I do have the same problem. How to install MOOSE with this forbidden error?

**Reply by milljm** [reply]
Not a solution, but can you peruse to the following sites, which were also moved over to cloudflare services? https://civet.inl.gov, https://mooseframework.inl.gov

**Reply by milljm** [reply]
@Aayush-T, Thank you for posting! I didn't read your entire post enough to realize the server was accessible only after authenticating as a human.

I am in contact with our Cloudflare team to try and get this "feature" disabled.

**Reply by milljm** [reply]
I have been told this was fixed yesterday at 14:30 MST.
